src/hotspot/share/compiler/compilerOracle.cpp

```diff
@@ -334,6 +334,10 @@ static void register_command(TypedMethodOptionMatcher* matcher,
     return;
   }
 
+  if (option == CompileCommandEnum::PrintCompilation) {
+    PrintCompilation = true;
```

Comment:
But this will set `PrintCompilation` to true globally, right? The directive should only apply to a specific method.

Reply:
Okey, I think we should close this PR.